apps/products-feed/src/modules/app-configuration/app-configurator.ts

```diff
@@ -0,0 +1,35 @@
+import { AppConfig } from "./app-config";
+import { SettingsManager } from "@saleor/app-sdk/settings-manager";
+
+export interface AppConfigurator {
+  setConfig(config: AppConfig): Promise<void>;
+  getConfig(): Promise<AppConfig | undefined>;
+}
+
+export class PrivateMetadataAppConfigurator implements AppConfigurator {
+  private metadataKey = "app-config";
+
+  constructor(private metadataManager: SettingsManager, private saleorApiUrl: string) {}
+
+  getConfig(): Promise<AppConfig | undefined> {
+    return this.metadataManager.get(this.metadataKey, this.saleorApiUrl).then((data) => {
+      if (!data) {
+        return data;
```

Comment:
```suggestion
        return undefined;
```

Reply:
its the same ;) but changed for the explicity

Reply:
I knew, but it threw me off at first because looked like a bug